src/Mod/Material/Resources/Materials/Standard/Thermoplast/PEEK-Generic.FCMat

```diff
@@ -0,0 +1,35 @@
+﻿---
+General:
+  UUID: "1fed8adb-7bf6-4d8a-9b1e-a3b50f8b2093"
+  Name: "PEEK-Generic"
+  Author: "FreeCAD"
+  License: "CC-BY-3.0"
+  Description: "Polyether ether ketone (PEEK) is a beige coloured organic thermoplastic polymer in the polyaryletherketone (PAEK) family, used in engineering applications. The values are average across the spectrum. If you need a more precise material definition use the more specialised PEEK-blend cards."
+  SourceURL: "https://www.matweb.com/search/DataSheet.aspx?MatGUID=2164cacabcde4391a596640d553b2ebe"
+Models:
+  Father:
+    UUID: "9cdda8b6-b606-4778-8f13-3934d8668e67"
+    Father: "Thermoplast"
+  Linear Elastic:
+    UUID: "7b561d1d-fb9b-44f6-9da9-56a4f74d7536"
+    Density: "1340 kg/mm^3"
```

Comment:
I'm guessing that this should be per m^3, not mm -- otherwise I think it's a touch high :grin: